napari/components/_tests/test_viewer_model.py

```diff
@@ -193,16 +193,19 @@ def test_mix_dims():
     assert viewer.dims.ndim == 3
 
 
-def test_new_labels():
-    """Test adding new labels layer."""
-    # Add labels to empty viewer
+def test_new_labels_empyty():
+    """Test adding new labels layer to empty viewer."""
     viewer = ViewerModel()
     viewer._new_labels()
     assert len(viewer.layers) == 1
     assert np.max(viewer.layers[0].data) == 0
     assert viewer.dims.ndim == 2
+    # Default shape when no data is present is 512x512
+    np.testing.assert_almost_equal(viewer.layers[0].data.shape, (512, 512))
```

Comment:
Almost?

Reply:
More specifically, 
```suggestion
    np.testing.assert_equal(viewer.layers[0].data.shape, (512, 512))
```

Reply:
almost equal is for floating point operations that may be subject to rounding/approximation error.